ohara-manager/client/src/components/Workspace/Edit/MoreActions.js

```diff
@@ -47,9 +129,32 @@ function MoreActions() {
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <MenuItem onClick={handleClose}>ADD PLUGINS</MenuItem>
-        <MenuItem onClick={handleClose}>ADD NODES</MenuItem>
+        <MenuItem onClick={() => setIsOpne(true)}>RESET WORKSPACE</MenuItem>
+        {/* Feature is disabled because it's not implemented in 0.9 */
+        false && <MenuItem onClick={handleClose}>ADD PLUGINS</MenuItem>}
+        {/* Feature is disabled because it's not implemented in 0.9 */
+        false && <MenuItem onClick={handleClose}>ADD NODES</MenuItem>}
       </Menu>
+      <Dialog
+        open={isOpen}
+        title={'Reset Workspace?'}
+        children={
+          <Typography>
+            Are you sure you want to reset the workspace, This action cannot be
+            undone!
+          </Typography>
+        }
+        confirmText={'RESET'}
+        handleClose={handleDialogClose}
+        handleConfirm={() => reset()}
```

Comment:
I don't see why we need to create another anonymous function here

```suggestion
        handleConfirm={reset}
```

Reply:
oh oh nice !!!